doc/latex.rst

```diff
@@ -783,6 +783,8 @@ Macros
      multiple paragraphs in header cells of tables.
   .. versionadded:: 1.6.3
      ``\sphinxstylecodecontinued`` and ``\sphinxstylecodecontinues``.
+  .. versionadded:: 3.0.0???
+     ``\sphinxkeypress``
```

Comment:
It seems `kbd` does not have a meaning for key "press". `\sphinxkeyboard` or `\sphinxkbd` might be better?
https://html.spec.whatwg.org/#the-kbd-element

Reply:
Changed to `\sphinxkeyboard`